### DISCUSSION TITLE
Altering the tilt of earth's axis

### DISCUSSION THREAD

**Discussion by cnighswonger** [Stellarium/stellarium#2072] [General]
Is it possible in Stellarium to alter the tilt angle of the earth's axis?

Chris

**Comment by gzotti** [comment]
Only when you are programming it. Else we try to model nature as good as necessary.

**Reply by cnighswonger** [reply]
If I were going to add a little code to inject an arbitrary value for epsilon for earth, would the place to do it be [here](https://github.com/Stellarium/stellarium/blob/564215bff6430a563690c4f55ed52880aa436635/src/core/planetsephems/precession.c#L262)?

Or better [here](https://github.com/Stellarium/stellarium/blob/master/src/core/modules/Planet.cpp#L1527)?

Maybe this question is better for the developer list?

**Reply by gzotti** [reply]
This is the place to ask. 

You can fork the project and modify as you wish in your private fork in the second place you pointed out.  Or rather in Planet::computeTransMatrix().

I am not interested in adding a deviation in the main project though.

**Reply by cnighswonger** [reply]
> You can fork the project and modify as you wish in your private fork in the second place you pointed out. Or rather in Planet::computeTransMatrix().

Thanks for the help!
 
> I am not interested in adding a deviation in the main project though.

No problem. This is only for some unique personal work in any case.
